From 8163343e97d68689ca7647dc738a9aa9ccb11b9d Mon Sep 17 00:00:00 2001
From: Guillaume SCIGALA <guillaume.scigala@essilor.fr>
Date: Sat, 22 Feb 2025 18:09:42 +0100
Subject: [PATCH] Modifications for cross-compile

---
 RaspberryPi_JetsonNano/c/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/RaspberryPi_JetsonNano/c/Makefile b/RaspberryPi_JetsonNano/c/Makefile
index 7767600..bc220e9 100644
--- a/RaspberryPi_JetsonNano/c/Makefile
+++ b/RaspberryPi_JetsonNano/c/Makefile
@@ -261,7 +261,7 @@ RPI:RPI_DEV RPI_epd
 JETSON: JETSON_DEV JETSON_epd
 
 TARGET = epd
-CC = gcc
+#CC = gcc
 MSG = -g -O -ffunction-sections -fdata-sections -Wall
 CFLAGS += $(MSG) -D $(EPD)
 
@@ -303,6 +303,6 @@ JETSON_DEV:
 	$(CC) $(CFLAGS) $(DEBUG_JETSONI) -c	 $(DIR_Config)/DEV_Config.c -o $(DIR_BIN)/DEV_Config.o $(LIB_JETSONI)  $(DEBUG)
 
 clean :
-	rm $(DIR_BIN)/*.* 
+#	rm $(DIR_BIN)/*.* 
 	rm $(TARGET) 
 
-- 
2.48.1

